lib/rubocop/cop/style/stabby_lambda_parentheses.rb

```diff
@@ -0,0 +1,92 @@
+# encoding: utf-8
+module RuboCop
+  module Cop
+    module Style
+      # Check for parentheses around stabby lambda arguments
```

Comment:
This should be extended a bit to feature some code examples. See other cops for details.